src/d3d/nvapi_d3d_low_latency_device.cpp

```diff
@@ -0,0 +1,58 @@
+#include "nvapi_d3d_low_latency_device.h"
+
+namespace dxvk {
+    bool NvapiD3dLowLatencyDevice::SupportsLowLatency(IUnknown* pDevice) {
+        auto d3dLowLatencyDevice = GetLowLatencyDevice(pDevice);
+        if (d3dLowLatencyDevice == nullptr)
+            return false;
+
+        return d3dLowLatencyDevice->SupportsLowLatency();
+    }
+
+    bool NvapiD3dLowLatencyDevice::LatencySleep(IUnknown* pDevice) {
+        auto d3dLowLatencyDevice = GetLowLatencyDevice(pDevice);
+        if (d3dLowLatencyDevice == nullptr)
+            return false;
+
+        return d3dLowLatencyDevice->LatencySleep();
```

Comment:
Since `d3dLowLatencyDevice->LatencySleep` returns `HRESULT`, we should  `return SUCCEEDED(...`.  This also applies to `SetLatencySleepMode`, `GetLatencyInfo` and `SetLatencyMarker`.